taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs

```diff
@@ -305,7 +280,6 @@ public override void RunCommand(object sender)
                                             where rw.Field<string>("Parameter Name") == "Attribute Name"
                                             select rw.Field<string>("Parameter Value")).FirstOrDefault().ConvertToUserVariable(sender);
 
-
                     var optionsItems = new List<string>();
                     var ele = (IWebElement)element;
                     var select = new SelectElement(ele);
```

Comment:
Please clean up **Script** namespace reference on line 298 and 383.

Reply:
done